packages/video_player/video_player_avfoundation/example/ios/RunnerTests/VideoPlayerTests.m

```diff
@@ -176,6 +178,250 @@ - (void)testSeekToInvokesTextureFrameAvailableOnTextureRegistry {
   XCTAssertEqual([player position], 1234);
 }
 
+- (void)testCanCacheMP4 {
+  NSObject<FlutterTextureRegistry> *mockTextureRegistry =
+      OCMProtocolMock(@protocol(FlutterTextureRegistry));
+  NSObject<FlutterPluginRegistry> *registry =
+      (NSObject<FlutterPluginRegistry> *)[[UIApplication sharedApplication] delegate];
+  NSObject<FlutterPluginRegistrar> *registrar = [registry registrarForPlugin:@"CantCachemp4"];
+  NSObject<FlutterPluginRegistrar> *partialRegistrar = OCMPartialMock(registrar);
+  OCMStub([partialRegistrar textures]).andReturn(mockTextureRegistry);
+  FLTVideoPlayerPlugin *videoPlayerPlugin =
+      (FLTVideoPlayerPlugin *)[[FLTVideoPlayerPlugin alloc] initWithRegistrar:partialRegistrar];
+
+  FlutterError *error;
+  [videoPlayerPlugin initialize:&error];
+  XCTAssertNil(error);
+
+  FLTIsCacheSupportedMessage *message = [FLTIsCacheSupportedMessage
+      makeWithUrl:@"https://flutter.github.io/assets-for-api-docs/assets/videos/hls/bee.mp4"];
+
+  FLTIsSupportedMessage *response = [videoPlayerPlugin isCacheSupportedForNetworkMedia:message
+                                                                                 error:&error];
+  XCTAssertNil(error);
+  XCTAssertTrue(response.isSupported.boolValue);
```

Comment:
Looks like it's just checking if `isCacheSupportedForNetworkMedia ` boolean, rather than checking whether it actually performs the caching (e.g. saving data to file, etc). 

How about renaming to something like: `testIsCacheSupported_yesForMP3`

Reply:
I've renamed the tests